File: 프로그래머스/0/120905. n의 배수 고르기/n의 배수 고르기.java
class Solution {
    public int[] solution(int n, int[] numlist) {
       
 int[] tempArray = new int[numlist.length];
        int count = 0;

        for (int num : numlist) {
            if (num % n == 0) {
                tempArray[count] = num;
                count++;
            }
        }

        
        int[] resultArray = new int[count];
        for (int i = 0; i < count; i++) {
            resultArray[i] = tempArray[i];
        }

        return resultArray;
    }
}

File: 프로그래머스/0/120905. n의 배수 고르기/n의 배수 고르기.js
function solution(n, numlist) {
    let tempArray = new Array(numlist.length);
    let count = 0;

    for (let num of numlist) {
        if (num % n === 0) {
            tempArray[count] = num;
            count++;
        }
    }

    
    let resultArray = new Array(count);
    for (let i = 0; i < count; i++) {
        resultArray[i] = tempArray[i];
    }

    return resultArray;
}